### PR TITLE
Fixing QT

### PR DESCRIPTION
**Pull request by nvarini**

### CODE REVIEW

File: paien.yaml
Comment (rmsds) [inline paien.yaml]: Do we need the `^mesa`, it seems to concretize just fine without it.

Also, as (at least) `glib` and `mesa` depend on Python we need to add an explicit dependency to one of our sanctioned Python installations, maybe:
```suggestion
      - qt@5.10.0 +opengl+webkit ^python@3.6.5+optimizations+tk
```

**Review by rmsds — CHANGES_REQUESTED** [review]
Besides my comment, we also need to verify all the not yet installed dependencies are added to the blacklist. From `spack spec -I qt@5.10.0 +opengl+webkit ^python@3.6.5+optimizations+tk %gcc@7.3.0 target=x86_E5v4_Mellanox`, I see:
```
glib
libmng
lcms
libtiff
mesa
expat
libbsd
glproto
libxdamage
damageproto
fixesproto
libxfixes
libxext
libxshmfence
libxv
videoproto
libxvmc
presentproto
py-mako
py-markupsafe
```

**Review by rmsds — APPROVED** [review]
Looks good to me.

And I'll wait for another review before merging because I don't know the use cases very well.
Pinging @alalazo and @nrichart

**Review by alalazo — APPROVED** [review]
Reminder to myself: in the next release we should probably blacklist implicits by default.

**Comment by alalazo** [comment]
Thanks @nvarini !

**Pull request merged**
